source/includes/stackpath/_predefined_edge_rules.md

```diff
@@ -0,0 +1,101 @@
+### Predefined Edge Rules
+
+The predefined edge rules let you configure how StackPath responds to requests to your website. These predefined rules only work with domains that resolve to StackPath.
```

Comment:
Should we rename `edge rules` to `EdgeRules`?

Reply:
852e6a9

Reply:
Updated different variations of `edge rules` by `EdgeRules` based on SP// docs and usage.
852e6a9